src/Search/FluentSearchVariant.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace TractorCow\Fluent\Search;
+
+use TractorCow\Fluent\Extension\FluentExtension;
+use TractorCow\Fluent\State\FluentState;
+use TractorCow\Fluent\Model\Locale;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\FullTextSearch\Search\Variants\SearchVariant;
+use SilverStripe\FullTextSearch\Search\SearchIntrospection;
+use SilverStripe\FullTextSearch\Search\Queries\SearchQuery;
+
+class FluentSearchVariant extends SearchVariant
+{
+
+    public function appliesToEnvironment()
+    {
+        return $this->config()->get('enabled');
+    }
```

Comment:
You could remove this, since `SearchVariant::appliesToEnvironment` is doing the same thing